WordPress/Classes/ViewRelated/System/ForcePopoverPresentationPopoverControllerDelegate.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+/// This delegate forces popover presentation even on iPhone / in compact size classes
+class ForcePopoverPresentationPopoverControllerDelegate: NSObject, UIPopoverPresentationControllerDelegate {
```

Comment:
I like the sound of that.

Perhaps we might also separate the two, with `ForcedPopoverPresenter` and separately an object that conforms to `UIPopoverPresentationControllerDelegate`. Maybe this way the `ForcedPopoverPresenter` would maintain a conforming object of something like `ForcedPopoverPresenterDelegate`, separating the presenter and delegate handler. I dunno, that may not be very swifty.